### PR TITLE
Fixed landing page for small screens.

### PR DESCRIPTION
**Pull request by AdityaPainuli**

### CODE REVIEW

**Comment by rohitdasu** [comment]
hi @AdityaPainuli i'm seeing that there is a page overflow(scroll bar) arising in the landing page. it was not there before. please fix it.

**Comment by AdityaPainuli** [comment]
Resolved the overflowing issue.

File: FE/pages/index.tsx [hunk 1]
```diff
@@ -54,12 +54,12 @@ const Home: NextPage = () => {
...
           <button
             type="button"
-            tw="mt-16 text-white bg-gray-800 hover:bg-gray-900 focus:outline-none focus:ring-4 focus:ring-gray-300 font-medium rounded-lg text-sm px-8 py-3 mr-2 mb-2 dark:bg-gray-800 dark:hover:bg-gray-700 dark:focus:ring-gray-700 dark:border-gray-700"
+            tw="lg:mt-16 mt-4 text-white bg-gray-800 hover:bg-gray-900 focus:outline-none focus:ring-4 focus:ring-gray-300 font-medium rounded-lg text-sm px-8 py-3 mr-2 mb-2 dark:bg-gray-800 dark:hover:bg-gray-700 dark:focus:ring-gray-700 dark:border-gray-700"
           >
             JOIN US
           </button>
         </div>
-        <div>
+        <div className="hidden lg:inline-flex">
```
Comment (rohitdasu) [inline FE/pages/index.tsx:62]: instead of className use tw. see other examples used in this component

**Review by rohitdasu — CHANGES_REQUESTED** [review]

**Review by rohitdasu — APPROVED** [review]
LGTM 👍

**Pull request merged**